sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -3,27 +3,89 @@
 
 #pragma once
 
+#include <string>
+#include <vector>
+
 #include <internal/contract.hpp>
 
+using namespace std;
+
 namespace azure
 {
 namespace core
 {
 namespace http
 {
 
-class http_request
+namespace http_method
 {
+enum HttpMethod
+{
+  GET,
+  HEAD,
+  POST,
+  PUT,
+  DELETE,
+  PATCH,
+};
+}
 
+class Header
```

Comment:
should't we be snake_case?

Reply:
thank God and @gilbertw , we have moved on to cammel case now 👍

Reply:
I thought we are still on snake case, am I wrong?

---
In reply to: [397557694](https://github.com/Azure/azure-sdk-for-cpp/pull/41#discussion_r397557694) [](ancestors = 397557694)

Reply:
oh please, why not snake_case :) I also haven't heard anything on camel/pascal casing.

Reply:
I asked during the day to Rick, but let's review it later, 
I am not really super against any style...  XD